src/main/java/com/github/complate/nashorn/renderer/NashornComplateRenderer.java

```diff
@@ -20,9 +20,17 @@
 import java.util.Map;
 import java.util.Optional;
 
-import static java.util.Collections.emptyMap;
```

Comment:
This import is required and because it was removed the code currently does not compile.